### ISSUE TITLE
Town types

### ISSUE THREAD

**Issue by ForgetfulDuran** [drewhoener/JWSuggestions#130]
I think there should be a few different town types
Ideas
Gated Commutity 
Requirements: Needs atleast 15 PREBUILT house plots 5 Shop plots (Must be 10x10) and 5 residents
Features of Gated Commuitity: Cheaper to start it (3.5k) (4.5k for Pro and lower) 2 Mayors NO PLOTS besides shop ones

Expensive Commuity 
Requirements: Needs atleast 15  35x35 or bigger houses and 10 shop plots and 10 residents and 50 plots
Features: Cheaper to start (3.0k) (4.0k for pro and lower) 1 Mayor and and a little honer

**Comment by ForgetfulDuran** [comment]
And there can be more

**Comment by 5d100** [comment]
I support this. it could be something for donators only. like, the highest form of community could only be for lords and then brackets.

i.e.
lords: Johnny Townhouses
epic & Ultimate : expensive community
pro - platinum - gated

**Comment by ForgetfulDuran** [comment]
No not like that
if you meet the requirements

**Comment by ForgetfulDuran** [comment]
But I like the sound of johnny townhouses

**Comment by 5d100** [comment]
it would give incentive to people to actually donate.

**Comment by drewhoener** [comment]
Yes, but it also requires towns to build themselves a certain way, structure their plots to be certain sizes and pre build plots in order to obtain these town ranks, if you want a town only for donators, then don't accept people who aren't donators

**Comment by 5d100** [comment]
you could have a town made by johnny or you where lords get their own plot. it would be something nice as an incentive, just an idea.
